### PR TITLE
Use mmap for utils.search_bytes_in_file.

### PR DESCRIPTION
**Pull request by oliverchang**
Renamed from utils.search_strings_in_file, and use mmap instead of attempting
to read the file line by line.

This also fixes a Python 3 exception.

### CODE REVIEW

**Comment by oliverchang** [comment]
(This is a bit of a risky change, so will test carefully before submitting).

File: src/python/tests/core/bot/fuzzers/utils_test.py [hunk 1]
```diff
@@ -12,73 +12,88 @@
...
+    shutil.rmtree(self.temp_dir, ignore_errors=True)
+
+  def _create_file(self, name, contents=b''):
+    path = os.path.join(self.temp_dir, name)
+    with open(path, 'wb') as f:
+      f.write(contents)
+
+    return path
 
   def test_not_a_fuzzer_invalid_name(self):
```
Comment (inferno-chromium) [inline src/python/tests/core/bot/fuzzers/utils_test.py]: Can you add a test where a file handle is passed (and not a string path)?

Comment (oliverchang) [reply src/python/tests/core/bot/fuzzers/utils_test.py]: Done!

**Pull request merged**